cypress/integration/mocked-okta/registerController.1.cy.ts

```diff
@@ -27,7 +27,7 @@ userStatuses.forEach((status) => {
             "Then I should be shown the 'Check your email inbox' page",
             () => {
               // Set the correct user status on the response
-              const response = { ...userResponse.response, status };
```

Comment:
This was changed as the actual behaviour in a non-mocked environment would be to create a user in the `STAGED` state

Reply:
Good point!